BackEndApp/v1/views.py

```diff
@@ -64,6 +64,7 @@ def generate(request):
     user_data = UserData(metadata, components, layers, inputs, preprocessing)
 
     status, error = parser.generate_code(inputs)
+    format_code("test.py")
```

Comment:
```suggestion
```

Reply:
This should be ideally called on line 76, just before `copy_file`. It will make sure, code if only called if the file is successfully created and thus not generating "File not found" errors.
I hope that makes sense :)

Reply:
Sure will do the changes.